test/service/test_manifest.py

```diff
@@ -132,11 +133,24 @@ def _get_manifest(self, format_: ManifestFormat, filters: Filters, stream=False)
         return requests.get(manifest.location, stream=stream)
 
     def _get_manifest_object(self, format_: ManifestFormat, filters: JSON) -> Manifest:
-        service = ManifestService(self.storage_service)
+        service = ManifestService(self.storage_service, self.file_url)
         return service.get_manifest(format_=format_,
                                     catalog=self.catalog,
                                     filters=filters)
 
+    def file_url(self,
```

Comment:
Any reason you can't use `self.app_module.app.file_url` here?

Reply:
No, I didn't think about it. Addressed it in the latest `!fixup`